src/Config/Configurations/Laptop.php

```diff
@@ -38,6 +39,7 @@ public static function v1(?ILoggerFactory $loggerFactory = null): Laptop
         $loggerFactory = $loggerFactory ?? new NullLoggerFactory();
         $grpcConfig = new StaticGrpcConfiguration(5000);
         $transportStrategy = new StaticStorageTransportStrategy($grpcConfig, $loggerFactory, self::$maxIdleMillis);
-        return new Laptop($loggerFactory, $transportStrategy);
+        $readConcern = ReadConcern::BALANCED;
+        return new Laptop($loggerFactory, $transportStrategy, $readConcern);
```

Comment:
Is this necessary given that the constructor defaults to read concern to null (balanced)?

Reply:
It's not necessary, but I added it for clarity. Can remove if you think that would be better.

I'd been referencing the [JS sdk](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/client-sdk-nodejs/src/config/configurations.ts#L76), which included it in the configs for documentation I think, but not actually necessary when computing whether to send the header (references: [node](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/client-sdk-nodejs/src/internal/cache-data-client.ts#L245-L248) and [web](https://github.com/momentohq/client-sdk-javascript/blob/44be3bfbfbe5ff2c562cfcb7a4cb12165de6e676/packages/client-sdk-web/src/internal/client-metadata-provider.ts#L43-L46)).